vsintegration/src/FSharp.Editor/LanguageService/MetadataAsSource.fs

```diff
@@ -99,7 +99,7 @@ module internal MetadataAsSource =
             ErrorHandler.ThrowOnFailure(windowFrame.SetProperty(int __VSFPROPID5.VSFPROPID_OverrideToolTip, name))
             |> ignore
 
-        windowFrame.Show() |> ignore
+        // windowFrame.Show() |> ignore
```

Comment:
Probably we should also remove all the dependencies on VS services like `SVsRunningDocumentTable`, and just create a Roslyn `Workspace` directly and work with that.

Reply:
We should. I didn't even try to since we eventually intend to move to lsp, I thought it wasn't worth to rewriting this part.